packages/blocks/src/surface-block/surface-block.ts

```diff
@@ -0,0 +1,13 @@
+import { BlockElement } from '@blocksuite/lit';
+import { customElement } from 'lit/decorators.js';
+
+import type { SurfaceBlockModel } from './surface-model.js';
+
+@customElement('affine-surface')
+export class SurfaceBlockComponent extends BlockElement<SurfaceBlockModel> {}
```

Comment:
Having a `render` method that returns `nothing` may be semantically better 🌚

Reply:
Done